lib/sprockets/commoner/bundle.rb

```diff
@@ -34,19 +44,37 @@ class Bundle < Schmooze::Base
       OUTRO = <<-JS.freeze
 }();
 JS
+      def initialize(root)
+        super(root, 'NODE_PATH' => JS_PACKAGE_PATH)
+      end
+
       def self.instance(env)
         @instance ||= new(env.root)
       end
 
       def self.call(input)
-        instance(input[:environment]).call(input)
+        env = input[:environment]
+        instance(env).call(input)
       end
 
       def call(input)
         return unless input[:metadata][:commoner_enabled]
+        env = input[:environment]
+        assets_in_bundle = Set.new(input[:metadata][:included]
+          .map { |uri| env.load(uri) }
+          .map { |asset| env.resolve(asset.filename, type: input[:content_type], pipeline: :self, compat: false)[0] })
+        assets_missing = Set.new(input[:metadata][:commoner_required]) - assets_in_bundle
```

Comment:
Ah yeah, I thought it could be nil if there's no files that set commoner_required, but looking at the code that's not the case